### ISSUE TITLE
Text Lenght Issue on Android

### ISSUE THREAD

**Issue by KaroNiko** [fcolecumberri/Doddler_Bizarre_Adventure_in_the_Matriarchy#5]
I specify that I am referring to the android version of xsystem35-sdl2, on other platforms I couldn't say. Here the text goes beyond the frame that should contain it, especially as regards the description of the various regions in the global map. This, however, rarely happens with regards to the dialogues, or the introductory description of the setting (as in the photo) seldom  going beyond the screen. Of course, all in all it's not a major problem. 

![1726042941390](https://github.com/user-attachments/assets/6dbc27f7-7922-4f89-bce5-6fc1c6c0184e)

I take this opportunity to recommend that dialogues be edited by a good expert in everyday spoken language (native english speaker), because as it stands the text is not sufficiently clear, coherent and fluid as a title of this magnitude deserves. Of course, the meaning of events can almost always be grasped, but the enjoyment of the game is severely compromised. I realize the project is in its early stages, but I hope someone with the right skills comes forward.
Thanks and Greetings.

**Comment by fcolecumberri** [comment]
> I specify that I am referring to the android version of xsystem35-sdl2, on other platforms I couldn't say. Here the text goes beyond the frame that should contain it, especially as regards the description of the various regions in the global map. This, however, rarely happens with regards to the dialogues, or the introductory description of the setting (as in the photo) seldom going beyond the screen. Of course, all in all it's not a major problem.

I am aware that menus and introductory description have text overflow issues (Both, android and desktop). That's still on my list of stuff to ether fix in code or change the text to a shorter version.

> I take this opportunity to recommend that dialogues be edited by a good expert in everyday spoken language (native english speaker), because as it stands the text is not sufficiently clear, coherent and fluid as a title of this magnitude deserves. Of course, the meaning of events can almost always be grasped, but the enjoyment of the game is severely compromised. I realize the project is in its early stages, but I hope someone with the right skills comes forward. Thanks and Greetings.

This also happens in Spanish (In which I am native), the problem is not the lack of a `native english speaker` the problem is the lack of people helping and the size of the project being massive, that's the main reason I made the GitHub page in hope of people that might help.

I'll left the issue open until I fix the text overflow.

**Comment by KaroNiko** [comment]
I take this opportunity to point out three typos in the first dialogues; a 95Tonomoki and a ORan (2 occurrences). 
Have you tried asking for help on Romhacking.net? Tom, Eien ni hen and other great, famed translators would perhaps be willing to collaborate; sure, the "hentai" parts could perhaps discourage them, but it would be enough not to mention and present those to them. 😅

**Comment by fcolecumberri** [comment]
The thing about ORan is that some characters call her that way, her name is Ran (蘭) but sometime people mention her as ORan お蘭 (Up to my understanding that's an honorific). Because of this I am not sure if I want to remove `ORan` → `Ran` or search for every appearance of お蘭 and make sure it got translated to ORan. I'll ask someone who understands Japanese better than me if I am not wrong.

**Comment by PookaMustard** [comment]
> The thing about ORan is that some characters call her that way, her name is Ran (蘭) but sometime people mention her as ORan お蘭 (Up to my understanding that's an honorific). Because of this I am not sure if I want to remove `ORan` → `Ran` or search for every appearance of お蘭 and make sure it got translated to ORan. I'll ask someone who understands Japanese better than me if I am not wrong.

Yeah that お is honorific and isn't a cutesy nickname (like Rin Kaenbyou's "Orin"). Alternatives would include calling her My Lady, or Lady Ran for example, and no need to directly translate every instance of 蘭 as Ran either. You should definitely not have "Oran".

Using these titles more often will also help better immerse the reader into the matriarchal world of the game.

**Comment by fcolecumberri** [comment]
Thanks @PookaMustard I will translate `お蘭` as `Lady Ran`. I hope to have a new patch before the year ends.

**Comment by PookaMustard** [comment]
For bonus points, as different characters interact with her, they might call her different titles. Ai could call her "Your highness" or something even more archaic to fit her "samurai" speech.

**Comment by fcolecumberri** [comment]
I would prefiere to alter the text the less I can to preserve it as close as possible to the original source. So if `お蘭` becomes `Lady Ran`, that would be for everyone.

Also if I don't do that, trying to keep remembering every possible character combination variation in Spanish and English will be impossible and eventually will become inconsistent.

**Comment by PookaMustard** [comment]
That's perfectly fine. You could consider it later, or even post-release. That said, you'll want to see the EN/ES lines logically flow from the previous and into the next line - so feel free to edit for coherence.

I haven't even been through February 昭和35 in-game but I'm already captivated by the untranslated game. Hope to play more soon.
